src/components/PopupWithForm.js

```diff
@@ -29,4 +31,12 @@ export default class PopupWithForm extends Popup {
         this._popupForm.reset();
         super.close();
     }
+
+    submitButtonState(submit, buttonText = "Saving...") {
```

Comment:
`submitButtonState` 

Function names should start with a **verb** in infinitive (there is such a point in the check list). It’s common in coding to do it so that functions could `speak` for themselves about what they do.



`set...`

Reply:
Changed to setSubmitButtonState